src/api/Families.ts

```diff
@@ -4,11 +4,47 @@ import API from '@/api'
 import { setToken } from '@/api/Auth'
 import { TFamily, IError, TFamilyFormPost } from '@/interfaces'
 
-export async function getFamilies(query: string | undefined | null) {
+export type TFamilySearchQuery = {
+  query?: string | null
+  geography?: string | null
+  status?: string | null
+  title?: string | null
+  description?: string | null
+}
+
+type TSearchParams = {
+  q?: string
+  geography?: string
+  status?: string
+}
+
+export async function getFamilies({
+  query,
+  geography,
+  status,
+}: TFamilySearchQuery) {
   setToken(API)
 
+  // For API performance reasons (search will timeout if too many results are returned)
+  // if there is no search, default to using 'redd'
```

Comment:
Comment to me for adding an exception for timeout in the API so Patrick can deal with it in the frontend.
https://linear.app/climate-policy-radar/issue/PDCT-633/add-timeout-exception-to-search-api